src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -123,6 +131,25 @@ public boolean updateUserDescription(User user, String description) {
     return persistentStorageAgent.updateUserDescription(existingUser.getName(), description);
   }
 
+  /**
+   * Verify {@code user} exists in {@code users}.
+   * If so, set new description for {@code user} from persistent storage,
+   * and return true if the operation is successful.
+   * Otherwise, return false.
+   *
+   * @param user          user whose account to update
+   * @param isAdmin       new isAdmin
+   *
+   * @return true if {@code user} description gets updated. Otherwise, false.
+   */
+  public boolean updateUserIsAdmin(User user, boolean isAdmin) {
+    if (!isUserRegistered(user.getName())) {
+      return false;
+    }
+    User existingUser = getUser(user.getId());
```

Comment:
Isn't this redundant? won't it just get the user that is already passed in?

Reply:
You're right!